exp-player/addon/components/exp-video-consent.js

```diff
@@ -13,7 +13,7 @@ export default ExpFrameBaseComponent.extend({
                 this.get('videoRecorder').on('onUploadDone', null);
                 this.sendAction('next');
             });
-            this.get('videoRecorder').start(`video-consent-${this.get('session.id')}`, this.$().find('.recorder'), {
+            this.get('videoRecorder').start(`video-consent-${this.get('session.id')}`, Em.$(this.element).find('.recorder'), {
```

Comment:
My PR for video play/recording has a fix for the video consent. I'm not sure if its the same issue.